Controller/Adminhtml/CompanyCodes/Get.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * @category    ClassyLlama
+ * @copyright   Copyright (c) 2018 Classy Llama Studios, LLC
+ * @author      sean.templeton
+ */
+
+namespace ClassyLlama\AvaTax\Controller\Adminhtml\CompanyCodes;
+
+use ClassyLlama\AvaTax\Exception\AvataxConnectionException;
+use Magento\Framework\DataObject;
+
+class Get extends \Magento\Backend\App\Action
+{
+    protected $resultPageFactory;
+
+    /**
+     * @var \ClassyLlama\AvaTax\Framework\Interaction\Rest\Company
+     */
+    protected $company;
+
+    /**
+     * @var \ClassyLlama\AvaTax\Helper\Config
+     */
+    private $config;
+
+    public function __construct(
+        \Magento\Backend\App\Action\Context $context,
+        \Magento\Framework\Controller\Result\JsonFactory $resultPageFactory,
+        \ClassyLlama\AvaTax\Framework\Interaction\Rest\Company $company,
+        \ClassyLlama\AvaTax\Helper\Config $config
+    )
+    {
+        parent::__construct( $context );
+        $this->resultPageFactory = $resultPageFactory;
+        $this->company = $company;
+        $this->config = $config;
+    }
+
+    public function execute()
+    {
+        $companies = [];
+        $postValue = $this->getRequest()->getPostValue();
+        $isProduction = (bool)$this->getRequest()->getParam( 'mode' );
+        $resultJson = $this->resultPageFactory->create();
+        $scope = isset( $postValue['scope'] ) ? $postValue['scope'] : null;
```

Comment:
@LordZardeck - There's no need to go back and change such instances in your AvaTax code, but just wanted to point out that your use of spaces around the values in parentheses in expressions isn't consistent with our general coding style.  I'd rather see the spaces eliminated.